packages/core/src/components/toggle/toggle.tsx

```diff
@@ -178,6 +178,10 @@ export class Toggle implements IxFormComponent<string> {
     }
     return (
       <Host
+        role="switch"
+        aria-label={toggleText}
```

Comment:
aria-label can be removed its not necessary for the accessibility tree
<img width="599" height="323" alt="image" src="https://github.com/user-attachments/assets/5e15cc74-8c25-40d0-8052-b405a8a8aedf" />